File: Api/Timesheet/Entities/EmployeePreferences.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Timesheet.Entities
{
    public class EmployeePreferences : EntityBase
    {
        public int EmployeeId { get; set; }
        public int TopList { get; set; }
        public string MenuFavorite { get; set; }

        [NotMapped]
        public int keyValue { get; set; }

        [NotMapped]
        public string keyName { get; set; }

    }
}

File: Web/src/app/models/records/employee-preferences.ts
export class EmployeePreferences {
  Id: number;
  EmployeeId: number;
  TopList: number;
  MenuFavorite: string;

  constructor(param: any) {
    this.Id = param.Id;
    this.EmployeeId = param.EmployeeId;
    this.TopList = param.TopList;
    this.MenuFavorite = param.MenuFavorite;
  }
}
